package.json

```diff
@@ -16,13 +16,15 @@
     "test-optimized": "node --allow-natives-syntax --trace_opt --trace_deopt test/optimized.js"
   },
   "dependencies": {
+    "@types/node": "^10.12.12",
```

Comment:
types should move to devDeps

Reply:
> types should move to devDeps

It's my first time to make PR, i think it will do better.😂😂

Reply:
It doesn't make sense for adding `@types/node` in `dependencies`. `dependencies` are required to run.